Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+
+# Install
+RUN apt-get -qq update
+RUN apt-get install -y curl
+RUN curl https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -
+RUN apt-get install -y clang-4.0 lldb-4.0 cmake
```

Comment:
Any reason why this uses clang-4.0?

The clang target in the original makefile is actually for 5.0: https://github.com/gcp/leela-zero/blob/7f9489ee1bc870ade3ee8c9db8a61d7977528304/src/Makefile#L19

Reply:
no particular reason, gcc works well in docker too

I will use the default gcc to save travis build time in next pr